### PR TITLE
Creating a TOC in readMe.md

### PR DESCRIPTION
**Pull request by kaverma**
Creating a TOC in readMe for links to contribution guidelines being published

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# Actions - GitHub Actions for Azure
+
+This repository provides a framework, guidleines and processes to author new and contribute to existing GitHub Actions deploying to Azure.
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- [GitHub Actions for Azure](#github-actions-for-azure)
```
Comment (N-Usha) [inline README.md:7]: This feels like a repetition of first line item

File: README.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- [GitHub Actions for Azure](#github-actions-for-azure)
+- [Contributing to Azure Actions](#contributing-to-azure-actions)
+  - [Create a new GitHub Action for Azure or Microsoft](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#creating-a-new-github-action-for-azure-or-microsoft)
+  - [Guidelines for repository permissions](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#guidelines-for-setting-permissions-on-the-repo)
+  - [Action Versioning](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#action-versioning)
+  - [Testing GitHub Actions](https://github.com/Azure/actions/blob/master/docs/Testing-GitHub-Actions.md)
+  - [Publish to Marketplace](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#publish-the-action-to-marketplace)
+  - [Communication guidelines](https://github.com/Azure/actions/blob/master/communication%20guidelines.md)
+- [Contributing to this repo](#contributing-to-this-repository)
```
Comment (N-Usha) [inline README.md:15]: Do we need this section again?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+- [Code of Conduct](#code-of-conduct)
```
Comment (N-Usha) [inline README.md:16]: Do we need a broader code of conduct for all the contributions across Azure Actions?

Comment (N-Usha) [reply README.md:16]: or have this as part of the TOC itself?

**Review by N-Usha — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kaverma) [reply README.md]: Changed

File: README.md (hunk 2 above)
Comment (kaverma) [reply README.md]: Yeah, removed this section as this content is in separate files

File: README.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
+# Actions - GitHub Actions for Azure
+
+This repository provides a framework, guidleines and processes to author new and contribute to existing GitHub Actions deploying to Azure.
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- [Existing GitHub Actions for Azure](#existing-github-actions-for-azure)
```
Comment (N-Usha) [inline README.md:7]: Can U just remove this line item? IMO the previous one is just enough

**Review by N-Usha — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (kaverma) [reply README.md:7]: Makes sense

File: README.md [hunk 5]
```diff
@@ -1,3 +1,17 @@
+# Actions - GitHub Actions for Azure
+
+This repository provides a framework, guidleines and processes to author new and contribute to existing GitHub Actions deploying to Azure.
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- Contributing to Azure Actions
+  - [Create a new GitHub Action for Azure or Microsoft](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#creating-a-new-github-action-for-azure-or-microsoft)
```
Comment (ds-ms) [inline README.md:8]: You could keep the path relative maybe? Instead of pointing to `master` branch

File: README.md [hunk 6]
```diff
@@ -1,3 +1,17 @@
...
+
+## Table of contents
+- [GitHub Actions for deploying to Azure](#github-actions-for-deploying-to-azure)
+- Contributing to Azure Actions
+  - [Create a new GitHub Action for Azure or Microsoft](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#creating-a-new-github-action-for-azure-or-microsoft)
+  - [Guidelines for repository permissions](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#guidelines-for-setting-permissions-on-the-repo)
+  - [Action Versioning](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#action-versioning)
+  - [Testing GitHub Actions](https://github.com/Azure/actions/blob/master/docs/Testing-GitHub-Actions.md)
+  - [Publish to Marketplace](https://github.com/Azure/actions/blob/master/docs/Process_of_Authoring_GitHub_Actions_for_Azure.md#publish-the-action-to-marketplace)
+  - [Communication guidelines](https://github.com/Azure/actions/blob/master/communication%20guidelines.md)
```
Comment (ds-ms) [inline README.md:13]: NIT: The naming convention of our files seem a little off to me

File: README.md [hunk 7]
```diff
@@ -10,7 +24,7 @@ To easily create GitHub CI/CD workflows targeting Azure, use our [Azure starter
 
 Please try out the [GitHub Actions for Azure](https://docs.microsoft.com/azure/developer/github/github-actions) and share your feedback via Twitter on [@Azure](https://twitter.com/azuredevops). If you encounter a problem, please open an issue on the GitHub repository for the specific action.
 
-# GitHub Actions for Azure
+# Existing GitHub Actions for Azure
```
Comment (ds-ms) [inline README.md:27]: `Official` maybe?

**Review by ds-ms — APPROVED** [review]
LGTM

File: README.md (hunk 6 above)
Comment (kaverma) [reply README.md:13]: Yeah, true, and for now not all the files are in the docs folder, will make that consistent

**Pull request merged**